internal/export/distro/messaging.go

```diff
@@ -0,0 +1,61 @@
+//
+// Copyright (c) 2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package distro
+
+import (
+	"github.com/edgexfoundry/go-mod-messaging/pkg/messaging"
+	"github.com/edgexfoundry/go-mod-messaging/pkg/types"
+)
+
+func InitMessaging(messageErrors chan error, messages chan *types.MessageEnvelope) {
+	// Create the messaging client
+	messageBus, err := messaging.NewMessageClient(types.MessageBusConfig{
+		SubscribeHost: types.HostInfo{
+			Host:     Configuration.MessageQueue.Host,
+			Port:     Configuration.MessageQueue.Port,
+			Protocol: Configuration.MessageQueue.Protocol,
+		},
+		Type: Configuration.MessageQueue.Type,
+	})
+
+	if err != nil {
+		LoggingClient.Error("failed to create messaging client: " + err.Error())
```

Comment:
Please use key/value style logging.

Reply:
k

Reply:
@brandonforster Please clarify. Some of these I don't know what you're looking for by adding another key. The above statement will log as `level=ERROR message=blah` among other fields.